### PR TITLE
chore: fixed the docker compose files for local testing

### PR DESCRIPTION
**Pull request by mkanoor**
The docker-compose files didn't have the default UI port of 8443 in the EVENT_STREAM_BASE_URL and EVENT_STREAM_MTLS_BASE_URL

### CODE REVIEW

**Review by Alex-Izquierdo — APPROVED** [review]

File: tools/docker/docker-compose-dev.yaml [hunk 1]
```diff
@@ -44,10 +44,8 @@ x-environment: &common-env
   EDA_SERVER_UUID: edgecafe-beef-feed-fade-decadeedgecafe
   EDA_PG_NOTIFY_DSN: "host=host.containers.internal port=5432 dbname=eda user=postgres password=secret"
   EDA_PG_NOTIFY_DSN_SERVER: "host=postgres port=5432 dbname=eda user=postgres password=secret"
-  EDA_EVENT_STREAM_BASE_URL: ${EDA_EVENT_STREAM_BASE_URL:-https://localhost/edgecafe-beef-feed-fade-decadeedgecafe/}
```
Comment (hsong-rh) [inline tools/docker/docker-compose-dev.yaml:47]: can we use `EDA_UI_PORT` instead of hard-coded 8443?

**Comment by jshimkus-rh** [comment]
@mkanoor For completeness the docker-compose-dev-redis-tls.yaml should possibly have these changes as well.

File: tools/docker/docker-compose-dev.yaml (hunk 1 above)
Comment (jshimkus-rh) [reply tools/docker/docker-compose-dev.yaml:47]: @hsong-rh You beat me to it. :-)

**Review by dhaustein — APPROVED** [review]

File: tools/docker/docker-compose-dev.yaml (hunk 1 above)
Comment (mkanoor) [reply tools/docker/docker-compose-dev.yaml:47]: @hsong-rh @jshimkus-rh I don't think we can nest one setting inside another setting.

**Pull request merged**